jakartaee/10/coreprofile/22.0.0.10-beta-Java17-TCKResults.adoc

```diff
@@ -0,0 +1,157 @@
+  `683e09ed991b61992a491857345cce635dd9e61a1f17663e2fb993c9d0f6b781`
++
+Jakarta Dependency Injection
+https://download.eclipse.org/ee4j/cdi/inject/2.0/jakarta.inject-tck-2.0.2-bin.zip[jakarta.inject-tck-2.0.2-bin.zip], SHA-256:
+  `23bce4317ca061c3de648566cdf65c74b57e1264d6891f366567955d6b834972`
++
+Jakarta JSON Binding
+https://download.eclipse.org/ee4j/jakartaee-tck/jakartaee10/promoted/eftl/jakarta-jsonb-tck-3.0.0.zip[jakarta-jsonb-tck-3.0.0.zip], SHA-256:
+  `954fd9a3a67059ddeabe5f51462a6a3b542c94fc798094dd8c312a6a28ef2d0b`
++
+Jakarta JSON Processing
+https://download.eclipse.org/ee4j/jakartaee-tck/jakartaee10/promoted/eftl/jakarta-jsonp-tck-2.1.0.zip[jakarta-jsonp-tck-2.1.0.zip], SHA-256:
+  `6ee953382ff965627fe20dd7e3bfce6c968ed829d611cf4990988ab54bfe8b54`
++
+Jakarta RESTful Web Services
+https://download.eclipse.org/ee4j/jakartaee-tck/jakartaee10/staged/eftl/jakarta-restful-ws-tck-3.1.1.zip[jakarta-restful-ws-tck-3.1.1.zip (Staged Release)], SHA-256:
+  `2ff33625931c5df8631a016bd25daabed9b27c81d72b7d2bfe6cab21b121cf1b`
+
+
+* Java runtime used to run the implementation:
++
+----
+openjdk version "17.0.3" 2022-04-19
+IBM Semeru Runtime Certified Edition 17.0.3.0 (build 17.0.3+7)
+Eclipse OpenJ9 VM 17.0.3.0 (build openj9-0.32.0, JRE 17 Linux amd64-64-Bit Compressed References 20220422_184 (JIT enabled, AOT enabled)
+OpenJ9   - 9a84ec34e
+OMR      - ab24b6666
+JCL      - dc07fd49b92 based on jdk-17.0.3+7)
+----
+
+* Summary of the information for the certification environment, operating system, cloud, ...:
++
+Apache Derby, Linux, Ubuntu 18.04
+
+* Challenges
++
+https://github.com/jakartaee/rest/issues/1126[RESTful Web Services TCK Challenge]
++
+https://github.com/jakartaee/rest/issues/1128[RESTful Web Services - excluded SEBootstrap tests]
+
+
+Test results:
+
+----
+Stage Name: Annotations TCK
+[javatest.batch] ********************************************************************************
+[javatest.batch] Completed running 1 tests.
+[javatest.batch] Number of Tests Passed      = 1
+[javatest.batch] Number of Tests Failed      = 0
+[javatest.batch] Number of Tests with Errors = 0
+[javatest.batch] ********************************************************************************
+[javatest.batch] PASSED........com/sun/ts/tests/signaturetest/caj/CAJSigTest.java#signatureTest
+
+
+Stage Name: CDI TCK
+[INFO] Tests run: 725, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 919.115 s - in TestSuite
+[INFO] 
+[INFO] Results:
+[INFO] 
+[INFO] Tests run: 725, Failures: 0, Errors: 0, Skipped: 0
+
+[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 4.35 s - in org.jboss.weld.langmodel.tck.LangModelTckTest
+[INFO] 
+[INFO] Results:
+[INFO] 
+[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0
+
+[INFO] /jakarta/conf/cdi-tck/target/surefire-reports/sigtest/TEST-liberty-cdi-tck-runner-4.0.5.xml: 0 failures in /jakarta/conf/cdi-tck/target/api-signature/cdi-api-jdk11.sig
```

Comment:
Is 4.0.5.xml right?  Should this be jdk17.sig?  Wrong result?  The 29.188 Total time below matches Java 11 which is suspicious.

Reply:
4.0.5 is because I've neglected to update the version of our runner pom to keep up with the galloping pace of CDI TCK development.  Farther up in the complete log, you would see things like this:
```
[INFO] Installing /jakarta/cdi-tck-4.0.7/artifacts/cdi-tck-parent-4.0.7.pom to /root/.m2/repository/jakarta/enterprise/cdi-tck-parent/4.0.7/cdi-tck-parent-4.0.7.pom
```

Update the total time, missed that one in my copy/paste/update cycle.

Reply:
Fixed in new commit.

Reply:
There is only one sig file in the CDI TCK for all JDK levels, but they left the jdk11 version tag on it for some reason.